src/index.js

```diff
@@ -1,41 +1,113 @@
 // import _ from 'lodash';
 import './style.css';
 
-const listContainer = document.querySelector('#todo-list');
-
-const LIST = [
-  {
-    description: 'Wash the dish',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Complete to app list project',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'To do project',
-    completed: false,
-    index: 2,
-  },
-
-];
-
-const loadTask = () => {
-  const sortedList = LIST.sort((item1, item2) => item1.index - item2.index);
-
-  for (let i = 0; i < sortedList.length; i += 1) {
-    listContainer.innerHTML += `
-      <div class="div-container"> 
-        <form class="form-desc">
-           
-         <input type="checkbox" class="checkbox" <p class="description-task">&nbsp&nbsp${sortedList[i].description}
-         <i class="fas fa-ellipsis-v pointer edit-task" aria-hidden="true"></i>
-        </form>
+class Task {
+  constructor() {
+    this.tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+    this.taskTitle = document.querySelector('#task-description');
+    this.taskStatus = document.querySelector('#author');
+    this.taskDisplay = document.querySelector('#display');
+    this.buttonAdd = document.querySelector('#add');
+    this.clearall = document.querySelector('#clearall');
+    this.checkboxStatus = document.querySelector('#checkboxid');
+    this.render();
+  }
+
+  addTask(id, title, status) {
+    this.tasks.push({ id, title, status });
+    localStorage.setItem('tasks', JSON.stringify(this.tasks));
+    this.render();
+  }
+
```

Comment:
Good job with adding and deleting tasks. But kindly add edit task functionality as requested on project requirements.
<img width="437" alt="Screenshot 2023-02-01 at 11 31 13 PM" src="https://user-images.githubusercontent.com/25887253/216156675-b3af7438-7bf9-49fa-a4b8-cef1d90ee2b3.png">